Build/packages.config

```diff
@@ -1,6 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Remotion.BuildScript" version="2.0.0-alpha024" targetFramework="net451" developmentDependency="true" />
+  <package id="DependDB.BuildProcessor" version="2.0.0" targetFramework="net451" />
+  <package id="DependDB.BuildProcessor.NuGetPreProcessor" version="2.0.0" targetFramework="net451" />
+  <package id="JetBrains.Annotations" version="2019.1.3" targetFramework="net451" />
+  <package id="MSBuild.Extension.Pack" version="1.5.0" targetFramework="net451" />
+  <package id="NUnit.ConsoleRunner" version="3.11.1" targetFramework="net451" />
+  <package id="NUnit.Extension.NUnitV2Driver" version="3.8.0" targetFramework="net451" />
+  <package id="NuGet.CommandLine" version="3.5.0" targetFramework="net451" developmentDependency="true" />
   <package id="NuGet.for.MSBuild" version="1.4.3" targetFramework="net451" developmentDependency="true" />
-  <package id="Remotion.BuildScript" version="2.0.0-alpha014" targetFramework="net451" developmentDependency="true" />
+  <package id="Remotion.BuildTools.MSBuildTasks" version="1.0.6862.28795" targetFramework="net451" developmentDependency="true" />
```

Comment:
packages.config in .nuget folder also needs updating.

Reply:
It has been deleted in the other projects, I guess we'll delete it here, too?

Reply:
there's two files, packages.config in build-project and packages.config in .nuget in root. As far as I can tell based on TypePipe, most dependencies are now located only in packages.config of build-project